boards/stm32mp157c-dk2/Makefile.include

```diff
@@ -0,0 +1,20 @@
+# we use shared STM32 configuration snippets
+INCLUDES += -I$(RIOTBOARD)/common/stm32/include
+
+# set default port depending on operating system
+PORT_LINUX ?= /dev/ttyACM0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
+
+# setup serial terminal
+include $(RIOTMAKE)/tools/serial.inc.mk
+
+# this board has an on-board ST-link adapter
+OPENOCD_CORE = "stm32mp15x.cm4"
+OPENOCD_CONFIG = board/stm32mp15x_dk2.cfg
+OPENOCD_ADAPTER_INIT = "-s ."
```

Comment:
Why is this needed?

Reply:
OPENOCD_CORE is to select the Cortex-M4 and not the default Cortex-A7.
OPENOCD_ADAPTER_INIT is an oversight